.github/workflows/ci.yml

```diff
@@ -1,6 +1,12 @@
 name: ci
 
-on: [pull_request, push]
+on:
+  pull_request:
+  push:
+    branches:
+      - "develop"
+      - "master"
+      - "release/*"
```

Comment:
Note: this change will stop us from running the ci workflow every time a push to _any_ branch happens. This will restrict the ci workflow to only the push to our main branches _and_ any time a PR is opened or updated. It should reduce unnecessary costs associated with running the workflow